src/autopas/molecularDynamics/LJFunctor.h

```diff
@@ -616,7 +619,8 @@ class LJFunctor
           device_handle.template get<Particle::AttributeNames::posZ>().get(),
           device_handle.template get<Particle::AttributeNames::forceX>().get(),
           device_handle.template get<Particle::AttributeNames::forceY>().get(),
-          device_handle.template get<Particle::AttributeNames::forceZ>().get());
+          device_handle.template get<Particle::AttributeNames::forceZ>().get(),
+          device_handle.template get<Particle::AttributeNames::ownershipState>().get());
```

Comment:
Can't we do this in a similar and also more future proof way as in `Functor::SoALoaderImpl` ?

Reply:
when we do the kokkos thing, maybe...